hw5.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+$n = 10;
+
+function Fibonachi($n)
+{
+    $Sum = 0;
+    $a = array();
+    $a [0] = 0;
+    $a [1] = 1;
+    if ($n == 1) 
+    {
+        $Sum = $a [0] = 0;
+    } elseif ($n == 2) 
+    {
+        $Sum = $a [0] + $a [1];;
+    } else {
+        for ($i = 2; $i < $n; $i++)
+        {
+            // $a[2] = $a[2-1] + $a[2-2];
+            // $a[2] = $a[1] + $a[0];
+            $a[$i] = $a[$i-1] + $a[$i-2];
+            $Sum = $Sum + $a[$i];
+        }
+    }
+    echo "Fibonachi sequence: ";
+    for ($i = 0; $i < $n; $i++)
+    {
+        echo $a[$i] . ", ";
+    }
```

Comment:
Для такої задачі можна використати вбудовану функцію
implode(", ", $a)
```suggestion
    echo implode(", ", $a);
```
https://www.php.net/manual/en/function.implode.php
також є прикольна функція яка навпаки з строки зробить массив
https://www.php.net/manual/en/function.explode.php
подивись їх на майбутнє, а так ми до них все одно дойдемо =)